src/basis.jl

```diff
@@ -84,6 +87,27 @@ function Base.unique(b::Basis)
     return Basis(b.basis[returns], variables(b), parameters = parameters(b))
 end
 
+function Base.unique(b₀::AbstractVector{Operation})
+    b = simplify_constants.(b₀)
+    N = length(b)
+    returns = Vector{Bool}()
+    for i ∈ 1:N
+        push!(returns, any([isequal(b[i], b[j]) for j in i+1:N]))
+    end
+    returns = [!r for r in returns]
+    return b[returns]
+end
+
+function fix_single_vars_in_basis!(basis,variables)
+    for (ind, el) in enumerate(basis)
+        for (ind_var, var) in enumerate(variables)
```

Comment:
I think here the enumerate can be change to `for var in variables`. The index does not matter.